worlds/yugioh06/rom.py

```diff
@@ -0,0 +1,163 @@
+import hashlib
+import math
+import os
+import struct
+
+from settings import get_settings
+
+import Utils
+from worlds.Files import APProcedurePatch, APTokenMixin, APTokenTypes
+
+from ..AutoWorld import World
```

Comment:
Hold up, did I miss this on the last review? This is supposed to be `from worlds.Autoworld import World`, we don't allow `..` imports from worlds